### DISCUSSION TITLE
Question: Can we specify the start version from which we would like to start tagging

### DISCUSSION THREAD

**Discussion by nnshet** [PaulHatch/semantic-version#97] [Q&A]
@PaulHatch We want start tagging from base number something like 5.0.0. Can we specify this value somewhere in the inputs?

**Comment by PaulHatch** [comment]
You can just add a tag to the first commit you want versioned or the last released version, the next run will pick it up and use it as the base for the next version.

Adding this as a config option has been requested before, but I have been hesitant to add it so far because I want to manage complexity of the config. You would you end up with a setting that is only used initially and does nothing at all for the rest of your project's lifetime. It might be added someday, but right now if no tag is present the version starts at 0.0.0.
